### PR TITLE
Add coverage for 2055790 to service enable disable

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
The first commit here is a cherry-pick of an old PR that I accidentally removed in one of my Foreman Maintain audits. I must not have had the latest changes on that branch. I'm adding that back in and also adding coverage for 2055790.

### CODE REVIEW

**Comment by Griffin-Sullivan** [comment]
trigger: test-robottelo
pytest: tests/foreman/maintain/test_service.py::test_positive_service_enable_disable

**Comment by Griffin-Sullivan** [comment]
trigger: test-robottelo
pytest: tests/foreman/maintain/test_service.py::test_positive_service_enable_disable

File: tests/foreman/maintain/test_service.py [hunk 1]
```diff
@@ -183,6 +188,28 @@ def test_positive_service_enable_disable(sat_maintain):
     result = sat_maintain.cli.Service.enable()
     assert 'FAIL' not in result.stdout
     assert result.status == 0
+    sat_maintain.power_control(state='reboot')
+    if type(sat_maintain) is Satellite:
```
Comment (JacobCallahan) [inline tests/foreman/maintain/test_service.py:192]: `isinstance` is prefered over `type` for type checking.
```suggestion
    if isinstance(sat_maintain, Satellite):
```

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
One minor change

**Comment by Griffin-Sullivan** [comment]
trigger: test-robottelo
pytest: tests/foreman/maintain/test_service.py::test_positive_service_enable_disable

**Review by sambible — APPROVED** [review]
PRT passing, looks good

**Pull request merged**